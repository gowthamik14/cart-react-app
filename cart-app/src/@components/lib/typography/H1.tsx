import React from 'react';

const H1: React.FC = ({ children }) => {
  return <span className="text-6xl">{children}</span>;
};

export default H1;
