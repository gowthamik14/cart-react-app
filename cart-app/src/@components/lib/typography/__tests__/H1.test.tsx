import React from 'react';
import { render } from '@testing-library/react';
import H1 from '@components/lib/typography/H1';

test('render H1', () => {
  
  const { container } = render(
    <H1/>
  );

  expect(container).toMatchSnapshot();
  
});
