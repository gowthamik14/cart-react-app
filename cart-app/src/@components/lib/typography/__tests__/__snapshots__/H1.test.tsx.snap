// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render H1 1`] = `
<div>
  <span
    class="text-6xl"
  />
</div>
`;
